package/yast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Mar 28 10:19:44 WEST 2017 - knut.anderssen@suse.com
+
+- backported
+  - SlideShow: Escape plain text release notes being shown properly
+    in RichText (bsc#1028721).
+- 3.1.215
```

Comment:
And it is a good opportunity to sync it with CASP, could we backport all the changes? I think that with the widgets will not be a problem, what about removing inserv #546 

@jreidinger @mchf could you check?, by now I can just backport this fix but was there some strong reason to only apply those changes to CASP branch?.